package.json

```diff
@@ -229,7 +229,7 @@
     "urlsafe-base64": "^1.0.0",
     "uuid": "^3.3.2",
     "valid-url": "^1.0.9",
-    "wd": "^1.11.3",
```

Comment:
Why are you updating these packages?

Reply:
I didn't update these, all I did was npm install.

Reply:
Reverted both package.json and package.lock.json files